game_patch/multi/multi.cpp

```diff
@@ -335,6 +337,10 @@ void multi_do_patch()
     multi_limbo_init.install();
     multi_start_injection.install();
 
+    // Allow server to customize dropped flag return timer in ms
+    AsmWriter(0x00473B88).push(g_additional_server_config.ctf_flag_return_time_ms); // blue flag
+    AsmWriter(0x00473B28).push(g_additional_server_config.ctf_flag_return_time_ms); // red flag
```

Comment:
```suggestion
    AsmWriter{0x00473B88}.push(g_additional_server_config.ctf_flag_return_time_ms); // blue flag
    AsmWriter{0x00473B28}.push(g_additional_server_config.ctf_flag_return_time_ms); // red flag
```

Reply:
@GooberRF could you apply this suggestion? I use curly braces in new code for calling constructors which is recommended in modern c++. I suppose remaining `AsmWriters` usages could be fixed in future to avoid this inconsistency